src/main/java/DAO/Exceptions/DAOException.java

```diff
@@ -0,0 +1,15 @@
+package dao.Exceptions;
```

Comment:
violation of package naming
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html